test/onnx/test_fx_op_consistency.py

```diff
@@ -1629,32 +1617,32 @@ def skip_torchlib_forward_compatibility(
         reason="Logic not implemented for size 0 inputs in op.Reshape",
         matcher=lambda sample: any(dim == 0 for dim in sample.input.shape),
     ),
-    xfail(
+    skip(
         "signal.windows.hamming",
         model_type=pytorch_test_common.TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM,
         reason="does not match node name",
     ),
-    xfail(
+    skip(
         "signal.windows.general_hamming",
         model_type=pytorch_test_common.TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM,
         reason="does not match node name",
     ),
-    xfail(
+    skip(
         "signal.windows.blackman",
         model_type=pytorch_test_common.TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM,
         reason="does not match node name",
     ),
-    xfail(
+    skip(
         "signal.windows.general_cosine",
         model_type=pytorch_test_common.TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM,
         reason="does not match node name",
     ),
-    xfail(
+    skip(
         "signal.windows.hann",
         model_type=pytorch_test_common.TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM,
         reason="does not match node name",
     ),
-    xfail(
+    skip(
```

Comment:
why changing xfail by skip?

Reply:
could use some help investigating this; for some reason if I left `xfail` then I was getting "unexpected success", but if I removed the xfail I was getting a failure…

Reply:
@liqunfu @titaiwangms @BowenBao was this the behavior you noticed this week too? what was the fix?

Reply:
> could use some help investigating this; for some reason if I left `xfail` then I was getting "unexpected success", but if I removed the xfail I was getting a failure…

@suo we will get back on this for you

Reply:
Put it down to the list SKIP_XFAIL_SUBTESTS with xfail should work. Ithink I met this kind of weird op tests before.